### DISCUSSION TITLE
0003 XLS-3d Standard for (sharing / deeplinking) signed transactions

### DISCUSSION THREAD

**Discussion by WietseWind** [XRPLF/XRPL-Standards#26] [XLS Proposals]
## The earlier version can be found by looking at the edit history.

**Comment by RichardAH** [comment]
I suggest we adopt a defacto standard for parsing like:
`<any url>signed-transaction:<SIGNEDBLOB>`
in the defacto standard `signed-transaction:<SIGNEDBLOB>` must always appear at the end of the URI

And a canonical standard (that developers should seek to adopt when generating URIs, rather than parsing them) like:
`<any url>?action=submit&tx=signed-transaction:<SIGNED BLOB>`

This allows some variation in future standards e.g.:
`<any url>?action=sign&tx=unsigned-transaction:<BLOB>`
etc.

Currently finishing up a new release for Toast so unless there are comments left here I'll be coding signed-tx parsing as `<any url>signed-transaction:<SIGNEDBLOB>`

**Comment by mvadari** [comment]
@WietseWind should this spec be moved into PR and kept in the repo or should it be closed? Note that this does not mean that the spec has been accepted/implemented/finalized.

I will move it to PR by copy-pasting the existing draft if there is no response in the next week.

**Reply by WietseWind** [reply]
Never really got the cross client support I hoped for. But keeping for record makes sense, hoping that someone one day someone will put it to good use. Reference implementation used in production here: https://www.npmjs.com/package/xumm-string-decode

**Reply by mvadari** [reply]
Would you mind opening a PR for it here? Easier if you do it than if I do it, you can add that link too 🙂

**Reply by sappenin** [reply]
Alternative is we can simply close very old conversations if the author doesn't have interest in pursuing a standard (there's still a record of closed convos in GH, FWIW). 

Not sure if you have a preference @WietseWind, but I would lean towards just closing unless you want to push a standard forward here.

**Reply by mvadari** [reply]
> there's still a record of closed convos in GH, FWIW

Yes, but it's a lot harder to find them (you have to specifically filter for closed discussions - others have recently had difficulty finding closed discussions). IMO if we want to keep it as a historical record (vs we don't care about it anymore), it should be a PR. EIPs allow `Draft`/`Stagnant` standards to be merged into the repo, and this feels like more of that category than something earlier in the pipeline.
